frontend/src/metabase/plugins/index.ts

```diff
@@ -287,3 +287,7 @@ export const PLUGIN_MODEL_PERSISTENCE = {
 export const PLUGIN_EMBEDDING = {
   isEnabled: () => false,
 };
+
+export const PLUGIN_CONTENT_VERIFICATION = {
+  VerifiedFilter: null as any,
```

Comment:
can we use something that's not `any`? Judging from `enterprise/frontend/src/metabase-enterprise/content_verification/components/VerifiedSearchFilter/VerifiedSearchFilter.tsx` I think this could be type as a React component if I'm not mistaken.